source/uwp/SharedVisualizer/ViewModel/HostConfigEditorViewModel.cs

```diff
@@ -73,5 +75,26 @@ public static async Task<HostConfigEditorViewModel> LoadAsync(MainPageViewModel
                 return new HostConfigEditorViewModel(mainPageViewModel);
             }
         }
+
+        protected static string PickHostConfigFile()
+        {
+            AccessibilitySettings accessibilitySettings = new AccessibilitySettings();
```

Comment:
should we also be setting [`HighContrastAdjustment` to `None`](https://learn.microsoft.com/en-us/windows/apps/design/accessibility/high-contrast-themes#setting-highcontrastadjustment-to-none)?

Reply:
I didn't make this change because `HighContrastAdjustment` handles all text colors by default.